spec/spec_helper.rb

```diff
@@ -85,7 +85,7 @@
   # end
 
   config.before(:each) do
-    EmsRefresh.debug_failures = true
+    EmsRefresh.debug_failures = true if defined?(EmsRefresh) && EmsRefresh.respond_to?(:debug_failures)
```

Comment:
@matthewd good point. Using "am I running migrations" makes sense. Using whether I happen to have loaded this modesl does not.